deploy/overlays/prod/patches/deployment-replicas.yaml

```diff
@@ -4,4 +4,4 @@ kind: Deployment
 metadata:
   name: mattermod
 spec:
-  replicas: 3
+  replicas: 1
```

Comment:
Depending on the result of the ongoing discussion, if this meant to be a temporary workaround this change is completely fine. But if mattermod has to run only in one instance this file (`deployment-replicas.yaml`) and the corresponding line in `prod/kustomization.yaml` should be deleted.